apps/coscrad-frontend/src/components/dynamicViews/commands/CommandPanel.tsx

```diff
@@ -1,12 +1,12 @@
 import { ICommandInfo } from '@coscrad/api-interfaces';
-import { ClassDataTypeMetadata } from '@coscrad/data-types';
+import { ClassSchema } from '@coscrad/data-types';
 import { CommandButton } from './CommandButton';
 
 export type CommandInfo = {
```

Comment:
Let's make sure this is not exposed through the public API anywhere.

Reply:
It isn't, but that means the front-end is currently broken. We will fix that as the next step.